client/src/views/uploadPage.vue

```diff
@@ -204,9 +183,12 @@ async function uploadCroppedImage() {
         <input
           type="file"
           ref="lunarImage"
-          accept="images/*"
+          accept="image/*"
           @change="onFileChange"
         />
+        <div v-if="data.fileSizeExceeded" id="status-message">
```

Comment:
this error prompt works! but can you make it a little bit more clear & obvious. Just add a style to it, make it bigger and make it red.

```
style="font-size:large; font-weight:bold; color:red;"
```